internal/nexodus/wg.go

```diff
@@ -111,7 +111,7 @@ func (ax *Nexodus) addPeerOS(wgPeerConfig wgPeerConfig) error {
 		Port: port,
 	}
 
-	interval := time.Second * 0
+	keepaliveInterval := time.Second * 20
```

Comment:
Look above in `addPeerUS()`. Can you change where it is using 5 seconds to match this use of 20? I would probably make it a constant at the top of the file so it's not repeated and accidentally gets out of sync later.

You can also drop the TODO comment above it.

Reply:
Note in that comment where it refers to docs that discuss keepalives. Those docs will need to be updated to reflect the changes in this PR.